config/routes.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  get 'home/index'
+  resources :messages
+  # get '/messages/random_greeting', to: 'messages#random_greeting'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      get 'messages', to: 'messages#index'
```

Comment:
Required
- To get the randomly generated message, the route should be to the corresponding controller method and in your case it's the random greeting method in the messages controller. Please fix the route here.